config/locales/en.yml

```diff
@@ -32,6 +32,7 @@ en:
           attributes:
             free_of_sanctions:
               blank: Tell us if your employment record is free of sanctions
```

Comment:
I think we might be able to get rid of this line now, since it's only an inclusion check?